pom.xml

```diff
@@ -19,7 +19,8 @@
   </description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Folders+Plugin</url>
   <properties>
-    <jenkins.version>2.46.3</jenkins.version>
+    <jenkins.version>2.60.1</jenkins.version>
+    <java.level>8</java.level>
```

Comment:
@jglick Cannot we use a more recent plugin-pom where this is defined?

Reply:
2.36 _is_ the most recent. We do not change the defaults for these two properties—plugins are expected to specify values.